litgpt/prompts.py

```diff
@@ -342,6 +349,8 @@ def model_name_to_prompt_style(model_name: str) -> PromptStyle:
         return Gemma()
     if re.search(r"OLMo.*-hf", model_name):
         return OLMo()
+    if re.search(r"Qwen2\.5-(?!Coder)", model_name):
```

Comment:
Why do not match Coder variant?

```python
from transformers import AutoTokenizer

prompt = "{prompt}"
messages = [{"role": "system", "content": "system_message"}, {"role": "user", "content": prompt}]

for model_name in ["Qwen/Qwen2.5-32B", "Qwen/Qwen2.5-32B-Instruct", "Qwen/Qwen2.5-Coder-32B-Instruct"]:
    tokenizer = AutoTokenizer.from_pretrained(model_name)
    text = tokenizer.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
    print(text)
```

produces identicall result for all three variants.

Reply:
You're right, thanks! I'll get it fixed.